compiler/jsgen.nim

```diff
@@ -39,7 +38,7 @@ from modulegraphs import ModuleGraph
 
 type
   TTarget = enum
```

Comment:
Is this really needed anymore? I'm not really into compiler code, but I think there's no point in this enum with only one possible value :)

Reply:
you are right ill remove it.